src/codecoverage/backend/tests/conftest.py

```diff
@@ -209,6 +209,21 @@ def coverage_builds():
     return builds
 
 
+def mock_coverage_by_changeset_job_success(job_changeset):
+    paths = glob.glob(os.path.join(FIXTURES_DIR, 'coverage_build_*.json'))
```

Comment:
This is replicating the same code as `coverage_builds`. Since you can't use the fixture here, you could move this code to a third, normal, function (`get_coverage_builds_fixtures`) which you call in both `coverage_builds` and `mock_coverage_by_changeset_job_success`.

Reply:
ah thats true, I didn't think of it. I'll update it!